frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -509,3 +509,14 @@ button.usa-pagination__button.usa-button {
 .usa-accordion__heading {
   border-bottom: 2px solid color(gray-warm-30);
 }
+
+#search-filter-drawer {
+  text-align: right;
+  .usa-modal-overlay {
+    padding: 0;
+    overflow: hidden;
+  }
+  .usa-modal__main {
+    width: 100%;
+  }
+}
```

Comment:
Simple solution with right alignment! 

We could even add a clever animation like… 

```
#search-filter-drawer {
  text-align: right;
  .usa-modal-overlay {
    padding: 0;
    overflow: hidden;
  }
  .usa-modal__main {
    width: 100%;
  }
  .usa-modal-wrapper {
    .usa-modal {
      transition: right 0.5s;
    }
    &.is-hidden .usa-modal {
      right: -100vw;
    }
    &.is-visible .usa-modal {
      right: 0vw;
    }
  }
}
```

Reply:
after playing with this for a bit, it looks more complicated - the truss modal is mainly handling the visibility of the modal with opacity and "visibility". The opacity we could transition but not so much the visibility. I like the idea of handling this with positioning instead but it'd be a bigger refactor from the built in modal implementation so I'd like to save that for future work